docs/eventing/broker/config-br-defaults.md

```diff
@@ -1,16 +1,51 @@
 ---
-title: "Default config (config-br-defaults)"
+title: "Default Broker ConfigMap"
 weight: 30
 type: "docs"
 ---
 
+Knative Eventing provides a `config-br-defaults` ConfigMap, which lives in the `knative-eventing` namespace, and provides default configuration settings to enable the creation of [multi-tenant Brokers](./_index.md) and InMemoryChannel type Channels.
```

Comment:
This sentence seems to overlap with the following paragraph ("Knative provides a `ConfigMap` which...")